packages/demo/src/App.svelte

```diff
@@ -269,7 +269,7 @@
     <button on:click={() => onboard.setChain({ chainId: '0x4' })}
       >Set Chain to Rinkeby</button
     >
-    <button on:click={() => onboard.setChain({ chainId: '0x89' })}
+    <button on:click={() => onboard.setChain({ chainId: '0x89', chainRpcUrl: 'https://polygon-rpc.com/' })}
```

Comment:
I couldn't find any tests in this repo, so I updated the demo. It won't do anything unless the version of the core package is published and updated correspondingly in the demo package.json.

Please let me know if there's any test that I could extend instead.